src/findbugs/findbugs-excludes.xml

```diff
@@ -23,4 +23,11 @@
     <!--Jenkins handles this issue differently or doesn't care about it.-->
     <Bug pattern="INFORMATION_EXPOSURE_THROUGH_AN_ERROR_MESSAGE"/>
   </Match>
+  <Match>
+    <Bug pattern="NM_METHOD_NAMING_CONVENTION"/>
```

Comment:
to generate with lowercase you need to change the properties to start with a lowercase, e.g. https://github.com/jenkinsci/jenkins/blob/96a5526bdb1bad3f553d9a9e7f5987a1e6b11456/core/src/main/resources/hudson/tools/Messages.properties#L34

it would break any external localization though and doesn't really matter